README.md

```diff
@@ -6,684 +6,42 @@
 
 # Mobile Center SDK for iOS
 
-## Introduction
+Mobile Center is mission control for mobile apps.
```

Comment:
Question: Is it better to use "Visual Studio Mobile Center" or "Microsoft Mobile Center", or simply "Mobile Center" as the title of the readme? I feel it's more clear to have "Visual Studio" as the official branding name here. Title of readme file is important because that's what contributors first see when they find this repository.